packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1256,16 +1356,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 electedSummarizerData ?? this.context.deltaManager.lastSequenceNumber,
                 SummarizerClientElection.isClientEligible,
             );
-            const summarizerClientElectionEnabled =
-                this.mc.config.getBoolean("Fluid.ContainerRuntime.summarizerClientElection") ??
-                this.runtimeOptions.summaryOptions?.summarizerClientElection === true;
-            const maxOpsSinceLastSummary = this.runtimeOptions.summaryOptions.maxOpsSinceLastSummary ?? 7000;
+            const isSummarizerClientElectionEnabled = this.summarizerClientElectionEnabled;
```

Comment:
nit: These 2 variables are not needed. You can directly use the properties from `this`.